.github/workflows/deploy-2-start.yml

```diff
@@ -89,6 +89,26 @@ jobs:
             spack.yaml \
             ${{ secrets.USER }}@${{ secrets.HOST_DATA }}:${{ vars.SPACK_YAML_LOCATION }}/${{ inputs.model }}.spack.yaml
 
+      - name: Fetch git sources for packages to build
+        # Workaround for ACCESS-NRI/spack#2.
+        # TODO: Remove once underlying spack issue is fixed
+        run: |
+          ssh ${{ secrets.USER }}@${{ secrets.HOST }} -i ${{ steps.ssh.outputs.private-key-path }} /bin/bash <<'EOT'
+
+          if [ ! -d "${{ steps.path.outputs.root }}/git_repos" ]; then
+            echo "::warning::Unable to find git_repos folder under ${{ steps.path.outputs.root }} - spack will attempt fetch instead"
+            exit 0
+          fi
+
+          cd "${{ steps.path.outputs.root }}/git_repos" || exit 1
```

Comment:
This exit condition won't b0rk the whole workflow though right?

Reply:
It will, actually. But I'll update it so it shouldn't. Technically that condition shouldn't even happen, since we checked above that the directory exists.